lib/livebook_web/components/app_components.ex

```diff
@@ -92,7 +92,9 @@ defmodule LivebookWeb.AppComponents do
   def deployment_group_form_content(assigns) do
     ~H"""
     <div>
-      <div class="flex flex-col">
+      <div class="flex flex-col gap-4">
+        <.text_field label="URL" type="base_url" field={@form[:url]} />
```

Comment:
I have added a `help=` label to this field in Teams, can you please copy it here?